packages/eas-cli/src/build/utils/url.ts

```diff
@@ -10,7 +10,9 @@ export function getProjectDashboardUrl(accountName: string, projectName: string)
 export function getBuildLogsUrl(build: BuildFragment): string {
   const { project } = build;
   if (project.__typename === 'App') {
-    return `${getExpoWebsiteBaseUrl()}/accounts/${project.ownerAccount.name}/builds/${build.id}`;
+    return `${getExpoWebsiteBaseUrl()}/accounts/${project.ownerAccount.name}/projects/${
```

Comment:
I know you haven't changed this in the PR but IMO it's better to use `new URL(...)` to format URLs:
https://nodejs.org/api/url.html#new-urlinput-base